playground/terraform/infrastructure/appengine/variables.tf

```diff
@@ -36,3 +36,13 @@ variable "location_id_us" {
   description = "Location of App"
   default = "us-central"
 }
+
+variable "location_id_eu" {
+  description = "Location of App"
+  default = "europe-west"
+}
+
+variable "feature_flag" {
```

Comment:
Could you add a description (and maybe a more descriptive name)? What is this a feature flag for?

Reply:
"feature_flag" was renamed to "skip_appengine_deploy".
AppEngine can be deployed only once per project, and if we need to deploy additional environment, AppEngine should not be enabled once again (of other case we will get an error from Terraform)